Purchases/Public/RCPurchases.h

```diff
@@ -415,11 +415,20 @@ NS_SWIFT_NAME(purchases(_:didReceiveUpdated:));
 
 /**
  Called when a user initiates a promotional in-app purchase from the App Store. If your app is able to handle a purchase at the current time, run the deferment block in this method. If the app is not in a state to make a purchase: cache the defermentBlock, then call the defermentBlock when the app is ready to make the promotional purchase. If the purchase should never be made, you don't need to ever call the defermentBlock and `RCPurchases` will not proceed with promotional purchases.
- 
+
  @param product `SKProduct` the product that was selected from the app store
  */
 - (void)purchases:(RCPurchases *)purchases shouldPurchasePromoProduct:(SKProduct *)product defermentBlock:(RCDeferredPromotionalPurchaseBlock)makeDeferredPurchase;
 
+
+/**
+ Updates the purchaser information.
+ This method should only be called when a purchase has been made through the RevenueCat server, but not through the app,
+ to force a refresh.
```

Comment:
@rkotzy can we get a better wording here? I don't think we need to mention the server and this could be use for other cases